### PR TITLE
Fix add to group action to strip group name before trying to look up

### PR DESCRIPTION
**Pull request by nicpottier**
Fixes a bug where someone was using a webhook result with a trailing space "Lahora ", and we were failing the lookup, but then creating a new group every time.

This adds a few methods to normalize names and normalize the looking up a group by name.

### CODE REVIEW

File: temba/api/views.py [hunk 1]
```diff
@@ -64,19 +64,19 @@ class WebHookEventReadView(WebHookEventMixin, SmartReadView):
...
             if obj.try_count == 3:
-                return "Never, three attempts errored, failed permanently"
+                return _("Never, three attempts errored, failed permanently")
             else:
                 if obj.status == 'C':
-                    return "Never, event delivered successfully"
+                    return _("Never, event delivered successfully")
                 else:
-                    return "Never, event deliverey failed permanently"
+                    return _("Never, event delivery failed permanently")
```
Comment (nicpottier) [inline temba/api/views.py]: Noticed this typo, then decided to add l18n here as well.

**Comment by nicpottier** [comment]
plz review

File: temba/flows/tests.py [hunk 2]
```diff
@@ -2272,7 +2272,19 @@ def test_group_actions(self):
         action.execute(run, None, sms)
 
         self.assertTrue(group.contacts.filter(id=self.contact.pk))
-        self.assertEquals(1, group.contacts.all().count())
+        self.assertEquals(group.contacts.all().count(), 1)
```
Comment (ericnewcomer) [inline temba/flows/tests.py]: Love the use of the s here.

Comment (nicpottier) [inline temba/flows/tests.py]: Gah!

**Pull request merged**